### DISCUSSION TITLE
Is there any way to specify `root directory` for a specific build?

### DISCUSSION THREAD

**Discussion by EricHasegawa** [vercel/next.js#61604] [Help]
### Summary

Trying to change the name of our root directory. AFAIK I can only specify one root directory, and since our production app is still using the old root directory name, I'm wondering if there's any way to deploy a preview or test the branch with the new root directory.

Furthermore, I'm wondering if the site will have downtime when I merge the name change and swap the root directory? Will there be some lag time where things could go down?

**Comment by ryanwhitemedia** [comment]
Bump. Having same issue. Should be an option in vercel.json IMO but it errors saying its not valid
